src/features/views/components/ViewDataTable/index.tsx

```diff
@@ -265,10 +270,14 @@ const ViewDataTable: FunctionComponent<ViewDataTableProps> = ({
       minWidth: 100,
       resizable: true,
       sortable: true,
-      width: 150,
+      width: configStr.fieldWidths[`col_${col.id}`] ?? 150,
       ...columnTypes[col.type].getColDef(col, accessLevel),
     })),
   ];
+  const { setColumnWidth } = useConfigurableDataGridColumns(
```

Comment:
This hook returns an object which includes the re-configured columns. It would have been possible to just do:

```ts
const { columns: gridColumns, setcolumnWidth } = useConfigurableDataGridColumns(
```

(And make the necessary changes to avoid the naming conflict of `gridColumns`).

This would have made all other changes in this PR unnecessary.